x-pack/packages/kbn-elastic-assistant-common/impl/schemas/chat/post_chat_complete_route.schema.yaml

```diff
@@ -0,0 +1,109 @@
+openapi: 3.0.0
+info:
+  title: Chat Complete API endpoint
+  version: '2023-10-31'
+paths:
+  /api/elastic_assistant/chat/complete:
+    post:
+      operationId: ChatComplete
+      x-codegen-enabled: true
+      description: Creates a model response for the given chat conversation.
+      summary: Creates a model response for the given chat conversation.
+      tags:
+        - Chat Complete API
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ChatCompleteProps'
+      responses:
+        200:
```

Comment:
Thoughts on capturing the chat/complete (non-stream) response interface here as well? 

I was going to comment and see if we wanted to return `conversationId` when `persist:true`, but then noticed we are just using the `ExecuteConnectorResponse` interface as part of `StaticReturnType`, which is what `langChainExecute` implicitly returns.

I guess more generally, what are your thoughts on chat/complete replacing `/actions/connector/{connectorId}/_execute` route? Would be nice to get away from that initial implementation that was mirroring the actions framework.